src/main/scala/com/supersonic/datadog/datadog_schemas.scala

```diff
@@ -21,7 +22,8 @@ case class TemplateVariable(name: String, prefix: String, default: Option[String
 object Graph {
   case class Definition(requests: List[Request],
                         visualization: Visualization,
-                        yAxis: Option[YAxis] = None)
+                        yAxis: Option[YAxis] = None,
+                        eventOverlay: EventOverlay)
```

Comment:
Please make this optional (with a default)

Reply:
Done